static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -135,6 +141,18 @@ class Dashboard extends Component<Props, State> {
     }
   }
 
+  fetchMemberList() {
+    const {api, selection} = this.props;
+    // Stores MemberList in MemberListStore for use in modals and sets state for use is child components
+    fetchOrgMembers(
+      api,
+      this.props.organization.slug,
+      selection.projects?.map(projectId => String(projectId))
+    ).then(_ => {
+      this.setState({memberList: MemberListStore.getAll()});
+    });
+  }
```

Comment:
Why not make the function async?

Reply:
Removed `.then`! Don't need to `async` `await` since we're relying only on the `reflux` store now, no longer need to store the `memberList` in state and pass to children through props.